driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -84,9 +85,9 @@ ExecutionInfo withIncomingPayload(Map<String, ByteBuffer> incomingPayload) {
      * <li>if a host is tried by the driver but is dead or in
      * error, that host is recorded and the query is retried;</li>
      * <li>on a timeout or unavailable exception, some
-     * {@link com.datastax.driver.core.policies.RetryPolicy} may retry the
```

Comment:
Nit: we usually do not add an import for a class that is only referenced through javadocs, to avoid creating a compile-time dependency between classes that are otherwise unrelated.